app/components/projects/settings/versions/related_issues_widget_component.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# -- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2010-2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+# ++
+
+module Projects
+  module Settings
+    module Versions
+      class RelatedIssuesWidgetComponent < Grids::WidgetComponent
+        option :version
+        option :issues, default: -> { [] }
+
+        MAX_DISPLAYED_ISSUES = 10
+
+        def title
+          I18n.t(:label_related_work_packages)
+        end
+
+        def call
+          return if issues.blank?
+
+          widget_wrapper do |widget|
+            render_issues(widget)
+            render_view_all_link(widget) if issues.size > MAX_DISPLAYED_ISSUES
+          end
+        end
+
+        private
+
+        def render_issues(widget)
+          issues.first(MAX_DISPLAYED_ISSUES).each do |issue|
+            widget.with_row do
+              helpers.flex_layout(flex_wrap: :nowrap, align_items: :center) do |flex|
+                flex.with_column(mr: 2) { issue.subject }
```

Comment:
It should be the other way around as we discussed in the channel. So firtst, the InfoLine, then the subject